xcm/xcm-builder/src/location_conversion.rs

```diff
@@ -345,6 +346,54 @@ impl<Network: Get<Option<NetworkId>>, AccountId: From<[u8; 20]> + Into<[u8; 20]>
 	}
 }
 
+/// Tinkernet ParaId used when matching Multisig MultiLocations.
+pub const TINKERNET_PARA_ID: u32 = 2125;
```

Comment:
This test is assuming your kusama para-id, right? it should be in the kusama runtime file, not in XCM-builder.

Reply:
Renamed the constants to include the KUSAMA_ prefix.